### DISCUSSION TITLE
TheRipper93's Carousel Combat Tracker Compatibility?

### DISCUSSION THREAD

**Discussion by YenBenGrey** [Anderware/Combat-Tracker-Extensions#3] [Q&A]
Hello there, 

This module looks amazing, do you know if it is compatible with Combat Carousel Module? 
If you are not sure, I think I might give it a test. :-)

**Comment by Anderware** [comment]
Hello, and thanks for your interest in this module!
I don't think it would be 100% compatible, but some base functionalities should be able to coexist.
